roles/openshift_health_checker/openshift_checks/etcd_traffic.py

```diff
@@ -0,0 +1,72 @@
+"""Module that scans journalctl for messages caused as a symptom of increased etcd traffic"""
+
+from openshift_checks import OpenShiftCheck, OpenShiftCheckException, get_var
+
+
+class EtcdTraffic(OpenShiftCheck):
+    """Check if host is being affected by an increase in etcd traffic."""
+
+    name = "etcd_traffic"
+    tags = ["health", "etcd"]
+
+    @classmethod
+    def is_active(cls, task_vars):
+        """Skip hosts that do not have etcd or masters in their group names."""
+        group_names = get_var(task_vars, "group_names", default=[])
+        valid_group_names = "masters" in group_names or "etcd" in group_names
+
+        version = get_var(task_vars, "openshift", "common", "version", default="0.0.0")
+        parsed_version = float(".".join(version.split(".")[0:2]))
+        valid_version = parsed_version == 3.4 or parsed_version == 3.5
```

Comment:
Thinking of versions as floats is just generally a dangerous thing to do.
In this case, we can just look at them as strings. For comparison, breaking out to integers would be the way to go (fortunately we don't put letters or unicode in our version strings).